contentcuration/contentcuration/frontend/channelList/views/ChannelSet/ChannelSetModal.vue

```diff
@@ -170,6 +170,7 @@
     },
     data() {
       return {
+        dialog: true,
```

Comment:
Temporarily emulate a usual non-dialog page behavior (until we can update these modal components to regular pages)